buildenv/jenkins/jobs/builds/Build-linux_x86-64

```diff
@@ -11,27 +11,33 @@ def setBuildStatus(String message, String state, String sha) {
     ]);
 }
 
+LABEL = (params.LABEL) ? "${params.LABEL}" : "Linux && x86"
+REPO = (params.OMR_REPO) ? "${params.OMR_REPO}" : "https://github.com/eclipse/omr.git"
+BRANCH = (params.OMR_BRANCH) ? "${params.OMR_BRANCH}" : ((params.MERGE_COMMIT) ? "${MERGE_COMMIT}" : "refs/heads/master")
+
 pipeline {
-    agent{label 'Linux && x86'}
+    agent{label "${LABEL}"}
     environment {
         PATH = "/usr/lib/ccache/:$PATH"
     }
     stages {
         stage('Get Sources') {
             steps {
                 timestamps {
-                    checkout([$class: 'GitSCM', branches: [[name: '${MERGE_COMMIT}']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/eclipse/omr.git']]])
-                    
-                    setBuildStatus("In Progress","PENDING","${MERGE_COMMIT}")
+                    checkout([$class: 'GitSCM', branches: [[name: "${BRANCH}"]], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[url: "${REPO}"]]])
+                    script {
+                        if(params.MERGE_COMMIT){
+                            setBuildStatus("In Progress","PENDING","${MERGE_COMMIT}")
+                        }
+                    }
                 }
             }
         }
         stage('Build') {
             steps {
                 timestamps {
                     echo 'Output CCACHE stats before running and clear them'
-                    sh '''ccache -s -z'''
-                    
+                    sh '''(ccache --version &>/dev/null && ccache -s -z) || echo "Warning: ccache not found"'''
```

Comment:
Why was this change made? There is no mention of this is the commit message.

Reply:
The Docker containers used by the internal Jenkins do not have `ccache` installed, which would cause this command to fail the job. Now if `ccache` is not found on a machine it will not fail, but skip past this command. I will update the commit message to include this explanation.